src/compas_robots/ghpython/scene/robotmodelobject.py

```diff
@@ -14,16 +14,14 @@ class RobotModelObject(GHSceneObject, BaseRobotModelObject):
 
     Parameters
     ----------
-    model : :class:`~compas_robots.RobotModel`
-        Robot model.
     **kwargs : dict, optional
         Additional keyword arguments.
         See :class:`~compas_ghpython.scene.GHSceneObject` and :class:`~compas_robots.scene.BaseRobotModelObject` for more info.
 
     """
 
-    def __init__(self, model, **kwargs):
-        super(RobotModelObject, self).__init__(model=model, **kwargs)
```

Comment:
Actually, if we're not doing anything in these constructors, can't we just delete them entirely? All 3 of them?

Reply:
Up to you, but they do have a useful docstring. The RobotModelObject for Blender do have a useful __init__, so that's maybe one reason to be consistent? 

Your call.

Reply:
Fair enough! "Clear docs" trumps "less code", let's keep them!